### PR TITLE
replaced tags with List[str]

### LINKED ISSUE #11: Replace tags as a list instead of dict
## 🚀 Feature Request

**💡 Got a brilliant idea?**
Loading tags as a dictionary is problematic for the frontend. As such, @siyangqiu proposes changing to a list of strings

**🤔 Is your feature request related to a problem?**
If your feature request is related to a problem you're facing, describe it here. The more we understand your struggle, the better we can address it.

**🎉 Describe the solution you'd like**
`tags (Dict[str, str], optional): Tags that can be used for grouping or sorting later. e.g. {"llm": "GPT-4"}.`
to
`tags (List[str], optional): Tags that can be used for grouping or sorting later. e.g. "GPT-4".`

**📚 Have you considered any alternatives?**
Status quo. Keep as dict.

**🔍 Additional context**
@siyangqiu please add additional thinking around this

### PR DESCRIPTION
**Pull request by areibman**
---
name: 📥 Pull Request
about: Contribute to Agentops by submitting a pull request
title: ''
labels: 'pull request'
assignees: ''
---

## 📥 Pull Request

**📘 Description**
Updated tags to be list[str] instead of dict[str, str]

**🔄 Related Issue (if applicable)**
If this PR is related to an existing issue, reference it here.

**🎯 Goal**
Easier for frontend

**🔍 Additional Context**
@siyangqiu to add context

**🧪 Testing**
Ran unit tests. Unit tests don't have type checking, so need to add Mypy + Coverage eventually.

### CODE REVIEW

File: agentops/agentops.py [hunk 1]
```diff
@@ -20,15 +20,15 @@ class AgentOps:
 
     Args:
         api_key (str): API Key for AgentOps services.
-        tags (Dict[str, str], optional): Tags for the sessions that can be used for grouping or sorting later (e.g. {"llm": "GPT-4"}).
+        tags (List[str], optional): Tags for the sessions that can be used for grouping or sorting later (e.g. []"GPT-4"}).
```
Comment (siyangqiu) [inline agentops/agentops.py]: Shouldn't this say [ "GPT-4" ]?

File: agentops/agentops.py [hunk 2]
```diff
@@ -89,12 +89,12 @@ def wrapper(*args, **kwargs):
...
         return decorator
 
-    def start_session(self, tags: Optional[Dict[str, str]] = None):
+    def start_session(self, tags: Optional[List[str]] = None):
         """
         Start a new session for recording events.
 
         Args:
-            tags (Dict[str, str], optional): Tags that can be used for grouping or sorting later. Examples could be {"llm": "GPT-4"}.
+            tags (List[str], optional): Tags that can be used for grouping or sorting later. Examples could be ['GPT-4']}.
```
Comment (siyangqiu) [inline agentops/agentops.py]: nit: we're using double quotes everywhere else for the List example

**Review by siyangqiu — APPROVED** [review]
Thanks for working on this! A few nits, but looks good!

File: agentops/agentops.py (hunk 1 above)
Comment (areibman) [reply agentops/agentops.py]: fixex

File: agentops/agentops.py (hunk 2 above)
Comment (areibman) [reply agentops/agentops.py]: Gonna have to set up a linting protocol. We're inconsistent

**Pull request merged**